packages/graph/src/graph2/graphT.ts

```diff
@@ -0,0 +1,1344 @@
+        const context = this;
+        const vertexElements = this._vertexG.selectAll<SVGGElement, VertexPlaceholder<V>>(".graphVertex");
+        const forceLabelOnHighlight = !context.showVertexLabels() && context.showVertexLabelsOnHighlight();
+        vertexElements
+            .classed("graphVertex-highlighted", d => !vertexMap || vertexMap[d.id])
+            .style("filter", d => vertexMap && vertexMap[d.id] ? "url(#" + this.id() + "_glow)" : null)
+            .each(function (d) {
+                if (forceLabelOnHighlight) {
+                    const props = context.calcProps(
+                        {
+                            showLabel: !!(vertexMap && vertexMap[d.id]),
+                            ...context.vertexMapper(d.props, d.props.origData)
+                        }
+                    );
+                    context._vertexRenderer(props, this);
+                }
+            })
+            .transition().duration(this.transitionDuration())
+            .on("end", function (d) {
+                if (vertexMap && vertexMap[d.id]) {
+                    if (d.element.node() && d.element.node().parentNode) {
+                        d.element.node().parentNode.appendChild(d.element.node());
+                    }
+                }
+            })
+            .style("opacity", function (d) {
+                if (d.props.hidden) return 0;
+                if (!vertexMap || vertexMap[d.id]) {
+                    return 1;
+                }
+                return context.highlight.opacity;
+            })
+            ;
+        return this;
+    }
+
+    highlightEdges(edgeMap) {
+        const context = this;
+        const edgeElements = this._edgeG.selectAll<SVGGElement, EdgePlaceholder<E, V>>(".graphEdge");
+        edgeElements
+            .classed("graphEdge-highlighted", function (d) { return !edgeMap || edgeMap[d.id]; })
+            .style("stroke-width", function (o) {
+                if (edgeMap && edgeMap[o.id]) {
+                    return context.highlight.edge;
+                }
+                return context.edgeStrokeWidth() + "px";
+            }).transition().duration(this.transitionDuration())
+            .style("opacity", function (o) {
+                if (o.source.props.hidden || o.target.props.hidden) return 0;
+                if (!edgeMap || edgeMap[o.id]) {
+                    return 1;
+                }
+                return context.highlight.opacity;
+            })
+            ;
+        return this;
+    }
+
+    highlightVertex(_element, d: VertexPlaceholder<V>) {
+        if (this.highlightOnMouseOverVertex()) {
+            if (d) {
+                const highlight = this.getNeighborMap(d);
+                highlight.vertices[d.id] = d;
+                this.highlightVerticies(highlight.vertices);
+                this.highlightEdges(highlight.edges);
+            } else {
+                this.highlightVerticies(null);
```

Comment:
VSC is complaining about this null. I guess because of the explicit type on highlightVerticies, unlike highlightEdges?